config.service.ts

```diff
@@ -66,5 +66,6 @@ export class HsConfig {
   sizeMode: string;
   locationButtonVisible: any;
   openQueryPanelOnDrawEnd: boolean;
+  dss_disallow_add: boolean; //found in datasources-metadata-dialog
```

Comment:
Wasnt this renamed to allowAddExternalDatasets ?

Reply:
Never mind, that was singleDatasources. To be honest i dont see a reason why the list should be presented at all if the user isn't able to add the layer to map. Maybe lets drop this ugly config parameter?